ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -354,7 +363,14 @@ public void shouldCloseServiceContextOnStop() {
 
   private void givenExecutorWillFailOnNoQueries() {
     standaloneExecutor = new StandaloneExecutor(
-        serviceContext, ksqlConfig, engine, queriesFile.toString(), udfLoader, true, versionCheckerAgent);
+        serviceContext,
+        processingLogConfig,
+        ksqlConfig,
+        engine,
+        queriesFile.toString(),
+        udfLoader,
+        true,
+        versionCheckerAgent);
```

Comment:
Maybe add a test to check that StandaloneExecutor calls `ProcessingLogServerUtils.maybeCreateProcessingLogTopic()` on start (and the analogous test for KsqlRestApplication)?